_layouts/topic.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: post
+---
+{% capture newline %}
+{% endcapture %}
+{% capture /dev/null %}
+  <!-- Get links for previous and next pages -->
+  {% assign topics_list = site.topics | natural_sort: "title" %}
+  {% capture first_link %}[{{topics_list[0].title}}]({{topics_list[0].url}}){% endcapture %}
+  {% capture last_link %}[{{topics_list[-1].title}}]({{topics_list[-1].url}}){% endcapture %}
+  {% for entry in topics_list %}
+    {% capture entry_link %}[{{entry.title}}]({{entry.url}}){% endcapture %}
+    {% if prev_link %}
+      {% assign next_link = entry_link %}
+      {% break %}
+    {% endif %}
+    {% if entry.url == page.url %}
+      {% assign prev_link = prev_entry | default: last_link %}
+    {% endif %}
+    {% assign prev_entry = entry_link %}
+  {% endfor %}
+
+  <!-- Build natural sentence-style list of aliases for this page's topic -->
+  {% if page.aliases != nil %}
+    {% assign num_aliases = page.aliases | size %}
+    {% capture aliases %}{:.center}{{newline}}*Also covering&nbsp;{% endcapture %}
+    {% if num_aliases > 2 %}
+      {% for alias in page.aliases %}
+        {% if forloop.last %}
+          {% capture aliases %}{{aliases}}, and {{alias}}{% endcapture %}
+        {% elsif forloop.first %}
+          {% capture aliases %}{{aliases}}{{alias}}{% endcapture %}
+        {% else %}
+          {% capture aliases %}{{aliases}}, {{alias}}{% endcapture %}
+        {% endif %}
+      {% endfor %}
+    {% else %}
+      {% capture aliases %}{{aliases}}{{page.aliases | join: ' and '}}{% endcapture %}
+    {% endif %}
+    {% assign aliases = aliases | append: '*' %}
```

Comment:
Should this line end with a `.` if it's meant to be a natural sentence?

Reply:
Because some of these are very short and because they're positioned special on the page, I think they look better without the period.